api/src/models/ImageFile.test.ts

```diff
@@ -0,0 +1,22 @@
+import ImageFile from "./ImageFile";
+import Config from "../models/Config";
+
+jest.mock("../models/Config");
+
+describe("Image", () => {
+    let image: ImageFile;
+    let config;
+    beforeEach(() => {
+        config = {
+            tesseractPath: '/usr/local/bin/tesseract',
```

Comment:
Same comment as above for these file paths.

Reply:
fixed